src/components/Templates/KnowledgeBaseTemplate.tsx

```diff
@@ -263,7 +261,7 @@ export default function KnowledgeBaseTemplate(
                   Related Reading
                 </Typography>
                 <div className={classes.menuSection}>
-                  {relatedDocuments.map(t => (
+                  {relatedDocuments.slice(0,10).map(t => (
```

Comment:
in the useRelatedDocuments hook it looks like we're filtering on all articles where there are any tags in common (line 37 in that file).  I think that additionally sorting by the highest # of tag matches would get us the most accurate related articles first